lib/apiservers/engine/backends/system.go

```diff
@@ -128,19 +99,14 @@ func (s *System) SystemInfo() (*types.Info, error) {
 		// These are system related.  Some refer to cgroup info.  Others are
 		// retrieved from the port layer and are information about the resource
 		// pool.
-		// FIXME: update this once we get the resource pool info
-		Name:            VchConfig().Name,
-		KernelVersion:   kernelVersion,         //stubbed
-		OperatingSystem: operatingSystem,       //stubbed
-		OSType:          platform.OSType,       //stubbed
-		Architecture:    platform.Architecture, //stubbed
-		NCPU:            runtime.NumCPU(),      //stubbed
-		MemTotal:        meminfo.MemTotal,      //stubbed
-		CPUCfsPeriod:    false,
-		CPUCfsQuota:     false,
-		CPUShares:       false,
-		CPUSet:          false,
-		OomKillDisable:  true,
+		Name:           VchConfig().Name,
+		KernelVersion:  "",
+		Architecture:   platform.Architecture, //stubbed
+		CPUCfsPeriod:   false,
```

Comment:
Well, I silenced them for the CLI, but API users need to beware.  Real solution is for us to submit a PR to Docker to ignore these value as they do for Windows once we GA.